aggregator/rpc_server.go

```diff
@@ -7,32 +7,61 @@ import (
 	"net/rpc"
 
 	"github.com/Layr-Labs/eigensdk-go/crypto/bls"
+	"github.com/Layr-Labs/eigensdk-go/logging"
 	blsagg "github.com/Layr-Labs/eigensdk-go/services/bls_aggregation"
 	taskmanager "github.com/Layr-Labs/eigensdk-go/task-manager"
 	sdktypes "github.com/Layr-Labs/eigensdk-go/types"
 	"github.com/Layr-Labs/eigensdk-go/utils"
 )
 
-// When starting the server, the aggregator start listening at the address specified by config the calls to
-// the ProcessSignedTaskResponse method
-func (agg *Aggregator[Input, Output]) startServer(ctx context.Context) error {
+// The aggregator RPC server is responsible for receiving signed responses from the operators, and sending
+// those responses to the BLS aggregation service.
+type AggregatorRpcServer[Input any, Output any] struct {
```

Comment:
Let's add a warning here that public methods added to this struct might be exposed to the network.